internal/internal.go

```diff
@@ -138,21 +138,21 @@ func (p *envoyExtAuthzGrpcServer) listen() {
 	// The listener is closed automatically by Serve when it returns.
 	l, err := net.Listen("tcp", p.cfg.Addr)
 	if err != nil {
-		logrus.WithField("err", err).Fatalf("Unable to create listener.")
+		logrus.WithField("err", err).Fatal("Unable to create listener.")
 	}
 
 	logrus.WithFields(logrus.Fields{
 		"addr":              p.cfg.Addr,
 		"query":             p.cfg.Query,
 		"dry-run":           p.cfg.DryRun,
 		"enable-reflection": p.cfg.EnableReflection,
-	}).Infof("Starting gRPC server.")
+	}).Debug("Starting gRPC server.")
```

Comment:
These logs don't have to be debug level (and they shouldn't be since they're important for knowing whether the listener started, with what configuration, etc.) Same w/ the exit log below.

Reply:
Changed log level to `Info`.